invenio_theme/assets/semantic-ui/less/invenio_theme/theme/elements/input.overrides

```diff
@@ -17,3 +17,7 @@
 
   }
 }
+
+.ui.disabled.input, .ui.input:not(.disabled) input[disabled] {
+  opacity: 1 !important;
```

Comment:
Brightens a bit the disabled input:
Before:
<img width="860" alt="Screenshot 2024-12-05 at 12 21 16" src="https://github.com/user-attachments/assets/7a787b03-d44d-4e28-8307-c52ebdd28c97">

After:
<img width="844" alt="Screenshot 2024-12-05 at 11 56 20" src="https://github.com/user-attachments/assets/796cdca5-5d81-4066-8feb-c774f0d2fb5f">